core/src/main/scala/org/broadinstitute/dsde/rawls/util/SentryEventFilter.scala

```diff
@@ -22,10 +24,24 @@ object SentryEventFilter {
                   if throwable.getMessage.contains("requester pays bucket but no user project provided") ||
                     throwable.getMessage.contains("billing account for the owning project is disabled") =>
                 null
+              // workbench-libs logs all error responses from Google APIs at ERROR level, including 4xx client errors.
+              // these would get picked up by Sentry, but we don't want them to. Check the status code returned by
+              // Google and filter out client errors.
+              //
+              // HttpResponseException covers GoogleJsonResponseException and TokenResponseException
+              case Some(googleResponse: HttpResponseException) if isStatusCodeIgnorable(googleResponse.getStatusCode) =>
+                null
+              // BaseHttpServiceException covers BigQueryException, ResourceManagerException, StorageException
+              case Some(googleService: BaseHttpServiceException) if isStatusCodeIgnorable(googleService.getCode) =>
+                null
               case _ =>
                 event
             }
           case _ => event
         }
     }
+
+  private def isStatusCodeIgnorable(code: Int): Boolean =
+    Option(code).nonEmpty && code > 0 && code / 100 < 5
```

Comment:
`code / 100 < 5` seems kinda weird... can't you just do like `code < 500` or something?  Or is there a good reason for this in particular?

Reply:
lol you're absolutely right. I [copy-pasted](https://github.com/broadinstitute/rawls/blob/fd092c70b4e228bee4e90a7b04d9267a1edfd4dc/core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala#L174) that and didn't stop to think about it - good catch!